web_01.html

```diff
@@ -147,7 +145,7 @@ <h1>Antic Popcorn</h1>
                         </div>
                         <div class="project__content">
                             <a href="#" class="project__image">
-                                <img src="/images/website_fonts.jpg" alt="">    
+                                <img src="/images/website_fonts.jpg"></img>   
```

Comment:
`img` tag doesn't have children and it is a self-closing tag similar to `<hr />`, `<link />` and etc hence it should look like this: `<img />`